example-testng-logback/pom.xml

```diff
@@ -21,10 +21,10 @@
     <dependencies>
 
         <dependency>
-            <groupId>com.epam.reportportal</groupId>
-            <!--<groupId>com.github.reportportal</groupId>-->
+<!--            <groupId>com.epam.reportportal</groupId>-->
+            <groupId>com.github.reportportal</groupId>
             <artifactId>agent-java-testng</artifactId>
-            <version>5.0.0-BETA-6</version>
+            <version>895e2b28</version>
```

Comment:
Guess that too outdated, please update dependencies.

Reply:
updated